src/Microsoft.ComponentDetection.Detectors/pip/IPyPiClient.cs

```diff
@@ -209,7 +209,7 @@ public async Task<PythonProject> GetProjectAsync(PipDependencySpecification spec
             {
                 var parsedVersion = PythonVersion.Create(release.Key);
                 if (release.Value != null && release.Value.Count > 0 &&
-                    parsedVersion.Valid && parsedVersion.IsReleasedPackage &&
+                    parsedVersion.Valid &&
```

Comment:
Similar logic is used in SimplePip, can you verify if it needs to be updated there too?